src/main/scala/no/ndla/draftapi/service/ConverterService.scala

```diff
@@ -693,16 +694,33 @@ trait ConverterService {
       (toKeep ++ updated).filterNot(_.isEmpty)
     }
 
-    def stateTransitionsToApi(user: UserInfo): Map[String, Seq[String]] = {
+    def _stateTransitionsToApi(user: UserInfo, article: Option[Article]): Map[String, Seq[String]] = {
+      val ignoreRequiredArchiveRoles =
+        article.exists(art =>
+          !art.status.current.equals(ArticleStatus.PUBLISHED) && !art.status.other.contains(ArticleStatus.PUBLISHED))
+
       StateTransitionRules.StateTransitions.groupBy(_.from).map {
         case (from, to) =>
           from.toString -> to
-            .filter(t => user.hasRoles(t.requiredRoles))
+            .filter(t => {
+              val ignoreRoles = ignoreRequiredArchiveRoles && t.from != ArticleStatus.PUBLISHED && t.to == ArticleStatus.ARCHIVED
```

Comment:
Er det overhodet sannsynlig at noen vil ønske seg tilsvarende for noen andre transitions?
I så fall stemmer jeg for å heller lage det som en regel i `StateTransitionRules` sånn som vi har gjort med andre ting